docs/dyndns/update-api.rst

```diff
@@ -100,3 +103,41 @@ Update Response
 If successful, the server will return a response with status ``200 OK`` and
 ``good`` as the body (as per the dyndns2 protocol specification). For error
 status codes, see above.
+
+
+Examples
+````````
+
+The examples below use ``mydomain.dedyn.io`` as the domain which is to be updated and 
+``mu4W4MHuSc0HyrGD1h/dnKuZBond`` as an API token affiliated with the respective account.
+(See :ref:`manage-tokens` for details.)
+
+
+Basic authentication with automatic IP detection (IPv4 **or** IPv6)::
+
+  curl --user mydomain.dedyn.io:mu4W4MHuSc0HyrGD1h/dnKuZBond https://update.dedyn.io
+
+Basic authentication with forced use of IPv6::
+
+  curl --user mydomain.dedyn.io:mu4W4MHuSc0HyrGD1h/dnKuZBond https://update6.dedyn.io
+
+Basic authentication with simultaneous update of IPv4 and IPv6::
+
+  curl --user mydomain.dedyn.io:mu4W4MHuSc0HyrGD1h/dnKuZBond \
+    https://update.dedyn.io/?myipv4=1.2.3.4&myipv6=fd08::1234
+
+REST API token authentication with automatic IP detection (IPv4 **or** IPv6)::   
```

Comment:
Let's not use the word REST here and below, as this is not a RESTful endpoint (it's just HTTP)